react-app-drawer-component/src/app-drawer.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+class AppMenuOptions extends React.Component {
+  render() {
+    const menuOptions = this.props.menuOptions;
+    return (
+      menuOptions.map(options =>
+        <p key={options.title} >{options.title}</p>
+      )
+    );
+  }
+}
```

Comment:
Since you are not using state or lifecycle methods, using a function component here would be better.